### ISSUE TITLE
Thonny can't handle UNC paths

### ISSUE THREAD

**Issue by aivarannamaa** [thonny/thonny#711]
```
Traceback (most recent call last):
   File "C:\Program Files (x86)\Thonny\lib\tkinter\__init__.py", line 1705, in __call__
     return self.func(*args)
   File "C:\Program Files (x86)\Thonny\lib\site-packages\thonny\code.py", line 601, in _cmd_save_file
     self.get_current_editor().save_file()
   File "C:\Program Files (x86)\Thonny\lib\site-packages\thonny\code.py", line 186, in save_file
     initialdir=get_workbench().get_cwd(),
   File "C:\Program Files (x86)\Thonny\lib\site-packages\thonny\workbench.py", line 1250, in get_cwd
     return normpath_with_actual_case(cwd)
   File "C:\Program Files (x86)\Thonny\lib\site-packages\thonny\common.py", line 212, in normpath_with_actual_case
     assert os.path.isabs(name)
AssertionError 
```

**Comment by aivarannamaa** [comment]
Aside: listing all network paths / shares: https://stackoverflow.com/questions/50528180/list-drives-and-network-locations-in-python-windows

**Comment by aivarannamaa** [comment]
Fixed in 3.2
